### PR TITLE
MTDSA-6692: Add template API code to new service.

### PR DESCRIPTION
**Pull request by BetaDraconis**

### CODE REVIEW

**Comment by BetaDraconis** [comment]
test

File: .scalafmt.conf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+align = more    // For pretty alignment.
```
Comment (BetaDraconis) [inline .scalafmt.conf:1]: test

File: app/config/FeatureSwitch.scala [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+package config
+
+import play.api.Configuration
+
+case class FeatureSwitch(value: Option[Configuration]) {
+
+  private val versionRegex = """(\d)\.\d""".r
+
+  def isWhiteListingEnabled: Boolean = {
```
Comment (david-walters-HMRC) [inline app/config/FeatureSwitch.scala:25]: isAllowListEnabled

File: app/config/FeatureSwitch.scala [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    value match {
+      case Some(config) => config.getOptional[Boolean]("white-list.enabled").getOrElse(false)
+      case None         => false
+    }
+  }
+
+  def whiteListedApplicationIds: Seq[String] = {
```
Comment (david-walters-HMRC) [inline app/config/FeatureSwitch.scala:32]: allowListedApplicationIds

**Pull request merged**